boards/st/steval_stwinbx1/doc/index.rst

```diff
@@ -0,0 +1,445 @@
+  - External memory interface supporting SRAM, PSRAM, NOR, NAND and FRAM memories
+  - 2 Octo-SPI memory interfaces
+
+- Rich analog peripherals (independent supply)
+
+  - 14-bit ADC 2.5-Msps, resolution up to 16 bits with hardware oversampling
+  - 12-bit ADC 2.5-Msps, with hardware oversampling, autonomous in Stop 2 mode
+  - 12-bit DAC, low-power sample and hold
+  - 2 operational amplifiers with built-in PGA
+  - 2 ultra-low-power comparators
+
+- Up to 22 communication interfaces
+
+  - USB Type-C / USB power delivery controller
+  - USB OTG 2.0 full-speed controller
+  - 2x SAIs (serial audio interface)
+  - 4x I2C FM+(1 Mbit/s), SMBus/PMBus
+  - 6x USARTs (ISO 7816, LIN, IrDA, modem)
+  - 3x SPIs (5x SPIs with dual OCTOSPI in SPI mode)
+  - 1x FDCAN
+  - 2x SDMMC interface
+  - 16- and 4-channel DMA controllers, functional in Stop mode
+  - 1 multi-function digital filter (6 filters)+ 1 audio digital filter with
+    sound-activity detection
+
+- CRC calculation unit
+- Development support: serial wire debug (SWD), JTAG, Embedded Trace Macrocell |trade|
+- True Random Number Generator (RNG)
+
+- Graphic features
+
+  - Chrom-ART Accelerator (DMA2D) for enhanced graphic content creation
+  - 1 digital camera interface
+
+- Mathematical co-processor
+
+ - CORDIC for trigonometric functions acceleration
+ - FMAC (filter mathematical accelerator)
+
+
+More information about STM32U585AI can be found here:
+
+- `STM32U585 on www.st.com`_
+- `STM32U585 reference manual`_
+
+Connectivity
+************
+
+   - **BlueNRG-M2SA** Bluetooth|reg| low energy v5.2 wireless technology module
+     (`BlueNRG-M2 datasheet`_)
+   - **MXCHIP EMW3080** (802.11 b/g/n compliant Wi-Fi module)
+   - **ST25DV64K** dynamic NFC/RFID tag IC with 64-Kbit EEPROM
+     (`st25dv64k datasheet`_)
+   - USB Type-C|trade| connector (power supply and data)
+   - STDC14 programming connector for **STLINK-V3MINI**
+     (`stlink-v3mini`_)
+   - microSD card socket
+
+Sensing
+*******
+
+  - **ILPS22QS** MEMS pressure sensor
+    (`ilps22qs datasheet`_)
+  - **STTS22H** Digital temperature sensor
+    (`stts22hh datasheet`_)
+  - **TSV912** wide-bandwidth (8 MHz) rail-to-rail I/O op-amp
+    (`tsv912 datasheet`_)
+  - **ISM330DHCX** iNEMO IMU, 3D accelerometer and 3D gyroscope with Machine Learning Core and Finite State Machine
+    (`ism330dhcx datasheet`_)
+  - **IIS3DWB** wide bandwidth accelerometer
+    (`iis3dwb datasheet`_)
+  - **IIS2DLPC** high-performance ultra-low-power 3-axis accelerometer for industrial applications
+    (`iis2dlpc datasheet`_)
+  - **IIS2MDC** 3-axis magnetometer
+    (`iis2mdc datasheet`_)
+  - **IIS2ICLX** high-accuracy, high-resolution, low-power, 2-axis digital inclinometer with Machine Learning Core
+    (`iis2iclx datasheet`_)
+  - **IMP23ABSU** analog MEMS microphone
+    (`imp23absu datasheet`_)
+  - **IMP34DT05** digital MEMS microphone
+    (`imp34dt05 datasheet`_)
```

Comment:
it looks like only stts22h has been configured in the devicetree, while at least a couple of the sensors in this list (ex. ISM330DHCX) are also supported in Zephyr. It would be nice to also add them to the board's dts.

Reply:
Will be added later.